app/editor/src/components/form/constants/FieldSize.ts

```diff
@@ -0,0 +1,7 @@
+export enum FieldSize {
```

Comment:
Now we should be able to control all form fields quickly if they use this!  It's pretty nice I think.